po/ru.po

```diff
@@ -1197,8 +1197,7 @@ msgstr "Артиллерист"
 msgid "Demolish"
 msgstr "Снести"
 
-#: data/base/messages/strings/names.txt:156
-#: po/custom/fromJson.txt:926
+#: data/base/messages/strings/names.txt:156 po/custom/fromJson.txt:926
```

Comment:
As you can see, Poedit joins source reference comment lines.
This can make reviewing patches much more cumbersome.
According to tucalipe, this behavior can be disabled by a setting in
Poedit's "File->Preferences->Advanced" tab.

Fortunately, this commit contains no changes to the translation.
Therefore, this can stay as it is right now.

Reply:
Thanks, I could not find this button, and for this half year did not offer these changes :(
Can you tell me how to return the translation to the old formatting style?

Reply:
> Thanks, I could not find this button, and for this half year did not offer these changes :(
> Can you tell me how to return the translation to the old formatting style?

I have installed Poedit and checked this myself.
Apparently, Tucalipe was wrong. The "Preferences" menu is part of the
"Edit" tab, not the "File" tab.

Inside the "Edit->Preferences->Advanced" menu, there is a checkbox
labelled "Wrap at 79" (not sure whether the number defaults to 79).
Yet if this box is not ticked, the source reference comment lines are
merged anyway :| It is probably best if I fix this with a shell script.
So far, I got rid of these merged lines by resetting hunks in git, with
either `git reset -p REVISION PO_FILE` or `git checkout -p PO_FILE`.

![poedit_edit_tab](https://user-images.githubusercontent.com/24465795/59350279-9ed27b00-8d13-11e9-8fbf-460a003a8a83.png)
![poedit_preferences_window](https://user-images.githubusercontent.com/24465795/59350293-a6921f80-8d13-11e9-8d7b-390190ccdf4f.png)